dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -0,0 +1,40 @@
+{{- $beatHome := printf "%s/%s" "/usr/share" .BeatName }}
+{{- $beatBinary := printf "%s/%s" $beatHome .BeatName }}
+
+FROM {{ .From }}
+
+LABEL \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.vendor="Elastic" \
```

Comment:
Community beats may want to reuse of image builders so this should be templated.

Reply:
Parameterized now.